packages/core/demo/data/combo.ts

```diff
@@ -0,0 +1,460 @@
+// Components
+import { SimpleBar } from "@carbon/charts/components/index";
+import { options } from "@carbon/charts/configuration";
+
+export const comboSimpleData = [
+	{ group: "School A", date: "Monday", value: 10000 },
+	{ group: "School A", date: "Tuesday", value: 65000 },
+	{ group: "School A", date: "Wednesday", value: 30000 },
+	{ group: "School A", date: "Thursday", value: 49213 },
+	{ group: "School A", date: "Friday", value: 49213 },
+	{ group: "Temperature", date: "Monday", temp: 70 },
+	{ group: "Temperature", date: "Tuesday", temp: 75 },
+	{ group: "Temperature", date: "Wednesday", temp: 31 },
+	{ group: "Temperature", date: "Thursday", temp: 31 },
+	{ group: "Temperature", date: "Friday", temp: 43 }
+];
+
+export const comboSimpleOptions = {
+	title: "Combo (Line + Simple bar) - custom configs",
+	axes: {
+		left: {
+			mapsTo: "value",
+			scaleType: "linear",
+			title: "USA Summer School Attendance"
+		},
+		right: {
+			mapsTo: "temp",
+			scaleType: "linear",
+			title: "Temperature (°F)",
+			correspondingDatasets: ["Temperature"]
+		},
+		bottom: {
+			title: "Day of the Week",
+			mapsTo: "date",
+			scaleType: "labels"
+		}
+	},
+	chartTypes: [
+		{
+			type: SimpleBar,
+			options: options.simpleBarChart,
+			datasets: [
+				"School A"
+			]
+		},
+		{
+			type: "line",
+			options: {
+				points: {
+					radius: 5
+				}
+			},
+			datasets: [
+				"Temperature"
+			]
+		}
+	]
+};
+
+export const comboHorizontalData = comboSimpleData;
+
+export const comboHorizontalOptions = {
+	title: "Horizontal Combo (Line + Simple bar)",
+	axes: {
+		top: {
+			mapsTo: "value",
+			scaleType: "linear",
+			title: "USA Summer School Attendance"
+		},
+		bottom: {
+			mapsTo: "temp",
+			scaleType: "linear",
+			title: "Temperature (°F)",
+			correspondingDatasets: ["Temperature"]
+		},
+		left: {
+			title: "Day of the Week",
+			mapsTo: "date",
+			scaleType: "labels"
+		}
+	},
+	chartTypes: [
+		{
+			type: "simple-bar",
+			options: {},
```

Comment:
what if `options` isn't included here? Does that cause an error?

Reply:
nope, you dont need it
if you dont need anything on top of the carbon charts styles that come with that chart you can put nothing or exclude it